From 4cb40606785c30710bcf39bba33167bee66ad0bd Mon Sep 17 00:00:00 2001
From: Dave Reisner <dreisner@archlinux.org>
Date: Mon, 8 Sep 2014 09:27:41 -0400
Subject: [PATCH] machined: remove redundant sd_notify

We already call this on via bus_event_loop_with_idle on exit. This
makes machined consistent with other similar daemons: localed,
hostnamed, timedated.
---
 src/machine/machined.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/machine/machined.c b/src/machine/machined.c
index f9d180d24a..aac670ba78 100644
--- a/src/machine/machined.c
+++ b/src/machine/machined.c
@@ -349,10 +349,6 @@ int main(int argc, char *argv[]) {
         log_debug("systemd-machined stopped as pid "PID_FMT, getpid());
 
 finish:
-        sd_notify(false,
-                  "STOPPING=1\n"
-                  "STATUS=Shutting down...");
-
         if (m)
                 manager_free(m);
 
